### PR TITLE
Make the repo complaint for REUSE

### PR DESCRIPTION
**Pull request by ghost**
The scope of this pull request is to make this repo complaint with a nice license notice specification called [REUSE](https://reuse.software) from the [FSFE](https://fsfe.org). Code and other files can be more distinguishable for us humans and machines regarding licensing.

More information is needed for complaince @MaddyUnderStars :
1. The SPDX specification distinguishes between `AGPL-3.0-only` and `AGPL-3.0-or-later`. (Just tagging `AGPL-3.0` is deprecated) Which one do you use in this repo? Both are compatible with each other.
`AGPL-3.0-or-later` has the advantage that you can use newer versions of the AGPL (if you want to) and can even specify a specific version `only` at a later point (if you want to). `AGPL-3.0-only` can create incompatibilities in the case there are newer versions published by the [FSF](https://fsf.org).
2. This `src/util/ghFIlePreview.ts` licensing isn't clear to me. Aluicord's license is incompatible though. Did you hack that code yourself?
(You can't steal code when you copy it, btw. It's mostly copyright infringement because of a license violation if it was unauthorized at all. I will pirate your computer on the sea for this, arrr. ;D )

### CODE REVIEW

**Comment by MaddyUnderStars** [comment]
yes algp3 or later
aliucords bot is not open source, no code was stolen. the comment is referencing the idea for the functionality.

File: .reuse/dep5 [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+ .gitignore
+ .husky/.gitattributes
+ .husky/pre-commit
+ .lintstagedrc
+ .prettierrc
+ package-lock.json
+ package.json
+ src/util/ghFIlePreview.ts
+ tsconfig.json
+Copyright: 2023 Maddy <https://github.com/MaddyUnderStars>
```
Comment (MaddyUnderStars) [inline .reuse/dep5]: Copyright should belong to the org rather than just me, probably

Comment (ghost) [reply .reuse/dep5]: Adding `2023 Spacebar Contributors <https://spacebar.chat>` to the copyright notes would be fine in my opinion.

**Comment by ghost** [comment]
`/docs-bot$ reuse lint`
image

**Comment by MaddyUnderStars** [comment]
One last thing, do you think you could add this to the CI and lintstagedrc? I'll forget, otherwise haha

**Comment by ghost** [comment]
I can try and look into it

**Comment by ghost** [comment]
I don't know what to do with lint-stage. You need the [reuse-tool](https://github.com/fsfe/reuse-tool) which is written in Python and do the check once with `reuse lint` in the root directory of the repo if you want to run it locally.

**Comment by MaddyUnderStars** [comment]
Alright, the CI is good enough. Thanks!

**Pull request merged**